### PR TITLE
community/py-sqlparse: upgrade to 0.2.4

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

**Comment by andypost** [comment]
@J0WI can you add a `check()`

**Comment by andypost** [comment]
Also you can modernize package before upgrade to remove `|| return 1` remains

**Comment by J0WI** [comment]
Done

File: community/py-sqlparse/APKBUILD [hunk 1]
```diff
@@ -2,22 +2,28 @@
...
-pkgrel=2
+pkgver=0.2.4
+pkgrel=0
 pkgdesc="A non-validating SQL parser module for Python"
 url="https://sqlparse.readthedocs.io"
 arch="noarch"
 license="BSD"
 depends=""
-makedepends="python2-dev py-setuptools python3-dev"
+makedepends="python2-dev py-setuptools pytest python3-dev"
```
Comment (andypost) [inline community/py-sqlparse/APKBUILD]: The `pytest` should be in `checkdepends` kinda https://github.com/alpinelinux/aports/pull/5476

Comment (J0WI) [reply community/py-sqlparse/APKBUILD]: Thanks, I've changed it!

File: community/py-sqlparse/APKBUILD [hunk 2]
```diff
@@ -20,6 +21,12 @@ build() {
 	python3 setup.py build
 }
 
+check() {
+	cd "$builddir"
+	python2 setup.py test
+	python3 setup.py test
```
Comment (andypost) [inline community/py-sqlparse/APKBUILD]: This should be 
```diff
-       python2 setup.py test
-       python3 setup.py test
+       python2 -m pytest .
+       python3 -m pytest .
```

Comment (J0WI) [reply community/py-sqlparse/APKBUILD]: I couldn't reproduce this locally, but changed it anyway.

**Comment by andypost** [comment]
Commited as b81fe602 5d9467cb 1c44ac7d

**Pull request closed without merging**